horreum-backend/src/test/java/io/hyperfoil/tools/horreum/svc/RunServiceTest.java

```diff
@@ -77,6 +92,52 @@ public void testDataSetCreated(TestInfo info) throws InterruptedException {
       }
    }
 
+   @org.junit.jupiter.api.Test
+   public void testStraightforwardTransformation(TestInfo info) throws InterruptedException {
+      BlockingQueue<DataSet> dataSetQueue = eventConsumerQueue(DataSet.class, DataSet.EVENT_NEW);
+      NamedJsonPath path = createExampleNamedJsonPath("foo", "$.value");
+      Schema schema = createExampleSchema(info);
+      assertNotNull(schema.id);
+      Transformer transformer = createExampleTransformerWithJsonPath("acme", schema, path);
+      Test exampleTest = createExampleTest(getTestName(info));
+      Test test = createTest(exampleTest);
+      overwriteTransformer(test, transformer);
+      uploadRun(schema, test.name);
```

Comment:
Are you intentionally using the schema object for Run.data?

Reply:
nope, replacing it with something more sensible.